provisioning/device/src/x509_registration.ts

```diff
@@ -32,11 +32,11 @@ export class X509Registration implements RegistrationClient {
         debug('security client returned error on cert acquisition');
         callback(err);
       } else {
-        this._transport.registerX509(registrationId, cert, forceRegistration, (err, result) => {
+        this._transport.registerX509(registrationId, cert, forceRegistration, (err, result, body) => {
```

Comment:
>body [](start = 92, length = 4)

not sure it's desirable to surface this at this point. it's ok to have it for debugging, we probably don't want to pass it to the callback though